test/test-applications/instrumentation/dependency-libs/Samples.ExampleLibrary/Class1.cs

```diff
@@ -0,0 +1,136 @@
+// <copyright file="Class1.cs" company="PlaceholderCompany">
```

Comment:
I think that everything under `test/test-applications/instrumentation/dependency-libs` is unused.

Moreover, I remember that @pjanotti was asking to reorganize in a way to have the productions code, test code, and sample code closer to each other.

Reply:
Not really unused. These are used for native helpers tests.

Reply:
The tests passed when I removed this directory

Reply:
I think these are running only under windows... as there are no linux tests.
See clr_helper_test.cpp

Reply:
I will check it tomorrow if you want. But I think that `test/test-applications/integrations/dependency-libs` was used by the tests

Reply:
Oh now I see what do you mean. `integrations` vs `instrumentations`, found the mistake and will fix it tomorrow.

Reply:
Removed.